### PR TITLE
feat: initWidget and drawChart & update `WidgetProps` type

### PR DESCRIPTION
**Pull request by yuda110**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci`, minor refactoring, etc.)
- [ ] Need discussion
- [x] Not that difficult
- [ ] Approved feature branch merge to master

### Description
Real Data is comming!!!!!!!!

스크린샷 2024-06-11 오후 10 29 07
스크린샷 2024-06-11 오후 10 29 23
스크린샷 2024-06-11 오후 10 31 22

### CODE REVIEW

File: apps/web/src/common/modules/widgets/_widgets/clustered-column-chart/ClusteredColumnChart.vue [hunk 1]
```diff
@@ -42,37 +62,105 @@ const state = reactive({
...
-                type: 'bar',
-                barMaxWidth: 24,
-                barGap: 0,
-                data: [150, 232, 201, 154, 190, 330, 410],
+        xAxis: {
+            type: 'category',
+            axisLabel: {
+                interval: 0,
+                formatter: (val) => {
+                    if (state.xAxisField === 'Date') {
```
Comment (sulmoJ) [inline apps/web/src/common/modules/widgets/_widgets/clustered-column-chart/ClusteredColumnChart.vue:70]: I would like to make the Date value a constant.

File: apps/web/src/services/dashboards/pages/DashboardPlaygroundPage.vue [hunk 2]
```diff
@@ -59,10 +68,12 @@ const handleClickAddWidget = () => {
             <div class="col-span-8">
                 <component :is="getWidgetComponent(state.selectedChartName)"
                            :widget-name="state.selectedChartName"
+                           :widget-options="state.sampleWidgetOptions"
                            widget-id=""
                            size="lg"
                            title=""
                            description=""
+                           data-table-id="asdfds"
```
Comment (sulmoJ) [inline apps/web/src/services/dashboards/pages/DashboardPlaygroundPage.vue:76]: ㅋㅋㅋㅋㅋㅋ

**Review by piggggggggy — APPROVED** [review]
Goooooood~

**Review by sulmoJ — APPROVED** [review]
LGTM

**Pull request merged**